src/components/tables/PoolsTable/PoolsTableActionsCell.vue

```diff
@@ -29,6 +30,11 @@ const { isMigratablePool } = usePool(toRef(props, 'pool'));
 /** COMPUTED */
 const stakablePoolIds = computed((): string[] => POOLS.Stakable.AllowList);
 const showVeBalLock = computed(() => isVeBalPool(props.pool.id));
+const isStakingBlocked = computed(() => {
+  const { id } = props.pool;
+
+  return POOL_MIGRATIONS_MAP[id].fromPoolId === id;
+});
```

Comment:
Let's not repeat this logic, instead let's just add a function to the `usePools` composable like:
```ts
function isStakingBlocked(pool: Pool): boolean {}
```

Reply:
I did in another way, just used `isMigratable` function from `usePool`. it is the same as `isStakingBlocked`